docs/installation/INSTALL_WINDOWS.md

```diff
@@ -86,20 +86,31 @@ in the wiki
 
 8. Start generating images!
 
-    # Command-line interface
-    (invokeai) python scripts/invoke.py
+    !!! Example ""
 
-    # or run the web interface on localhost:9090!
-    (invokeai) python scripts/invoke.py --web
+        === "CLI"
```

Comment:
This whole block doesn't appear to format correctly when previewed.

Reply:
are you using MkDocs to preview?

Reply:
<img width="771" alt="image" src="https://user-images.githubusercontent.com/40327258/200053323-d14fe3e8-5cf9-4de8-9276-3ff3ece0cd34.png">

Reply:
(Screenshot did not show up in VSCode for me)

Reply:
Nope, just looking at the files on GitHub. Is there a way to make the GitHub markdown cleaner and still able to generate MkDocs from it that look nice?

Reply:
This is how they all look on GitHub:
![image](https://user-images.githubusercontent.com/1947631/200055251-cd1b43d9-96ad-4623-b486-714073f80783.png)